.github/workflows/triage.yml

```diff
@@ -32,34 +32,27 @@ jobs:
         uses: xom9ikk/dotenv@v1.0.2
         with:
           path: .github/workflows/
-      - name: Preparing string variables
-        run: |
-          TRIAGE_TEAM_STRING=${TRIAGE_TEAM/[/}
-          TRIAGE_TEAM_STRING=${TRIAGE_TEAM_STRING/]/}
-          TRIAGE_TEAM_STRING=${TRIAGE_TEAM_STRING//'"'/}
-          # creating env variable "on the fly"
-          echo "TRIAGE_TEAM_STRING=$TRIAGE_TEAM_STRING" >> $GITHUB_ENV
       - name: Assign to a person to work on it
         uses: pozil/auto-assign-issue@v1.7.3
         with:
           numOfAssignee: 1
-          assignees: ${{ env.TRIAGE_TEAM_STRING }}
+          #assignees: ${{ env.TRIAGE_TEAM_STRING }}
```

Comment:
Could we remove it?

Reply:
Ok, I left it as #teams was. But it's true, let's remove it.